config/prod/gates-larssono-momi-ec2.yaml

```diff
@@ -0,0 +1,47 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-linux-v1.j2
+stack_name: gates-larssono-momi-ec2
+sceptre_user_data:
+  Distro: "ubuntu"     # (valid values: aws, ubuntu) AMIId must match distro
+  # (Optional) Expose ports to incoming traffic (default is no open ports) (valid range: 1-65535)
+  OpenPorts: ["80","443"]
```

Comment:
you only need to enable this for public instances.  All ports are already open for private instances once a user is logged into our VPN